profiles/ayushmangarg2003.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/105537793/188279342-010cebd6-71f7-4fb3-9a5e-1e49528b1f4c.png",
+  "issueId": 66,
```

Comment:
Hey, raise your issue and then put that issue number here. If you need help feel free to pin us.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Their issue seems to be #1359.
```suggestion
  "issueId": 1359,
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix